code/modules/mining/equipment/mining_tools.dm

```diff
@@ -47,32 +50,38 @@
 
 /obj/item/pickaxe/silver
 	name = "silver-plated pickaxe"
+	desc = "A silver-plated pickaxe that mines slightly faster than standard-issue."
 	icon_state = "spickaxe"
 	inhand_icon_state = "spickaxe"
 	worn_icon_state = "spickaxe"
 	toolspeed = 0.5 //mines faster than a normal pickaxe, bought from mining vendor
-	desc = "A silver-plated pickaxe that mines slightly faster than standard-issue."
-	force = 17
+	force = 18
 
 /obj/item/pickaxe/diamond
 	name = "diamond-tipped pickaxe"
+	desc = "A pickaxe with a diamond pick head. Extremely robust at cracking rock walls and digging up dirt."
 	icon_state = "dpickaxe"
 	inhand_icon_state = "dpickaxe"
 	worn_icon_state = "dpickaxe"
 	toolspeed = 0.3
-	desc = "A pickaxe with a diamond pick head. Extremely robust at cracking rock walls and digging up dirt."
-	force = 19
+	force = 20
```

Comment:
can these break open lockers now?

Reply:
Yes, so can thrown spears and those are even easier to get.

Reply:
Yeah keep in mind that diamond pickaxes are only available from the mining rewards vendor, and while a busy miners might be able to buy 15 of them in a round, they do take a reasonable amount of work to get a hold of.